.yamato/upm-ci.yml

```diff
@@ -1,7 +1,5 @@
 editors:
-  - version: 2018.3
   - version: 2019.1
-  - version: trunk

```

Comment:
NOTE: I pulled trunk out of Yamato config so that we can stabilize tests on 2019.1